docs/integrator/recipe-development.md

```
@@ -535,25 +557,25 @@ spec:
     - name: gpu-operator
-      version: v26.3.2
       valuesFile: components/gpu-operator/eks-gb200-training.yaml
 ```
 
 ### Updating Recipes
 
-**Updating versions:**
+**Updating versions:** bump the component's registry default in
+`recipes/registry.yaml` — overlays inherit it, so no overlay edit is needed
+(see [Chart Version Pinning](#chart-version-pinning)):
 ```yaml
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)